pkg/translator/jaeger/jaegerproto_to_traces.go

```diff
@@ -27,6 +27,7 @@ import (
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/ptrace"
 	conventions "go.opentelemetry.io/collector/semconv/v1.6.1"
+	conventions1_9 "go.opentelemetry.io/collector/semconv/v1.9.0"
```

Comment:
I would like some guidance on this part. Could we just migrate to `v1.9.0`?

Reply:
Ive no exp with that. @Aneurysm9, @jpkrohling could you give some input (noticed you updated the documentation in the past)?

Reply:
If `v1.6.1` does not contain the required attribute then migrate to a version that does.  There should generally only be one version of the semantic conventions imported into any given module.

Reply:
I updated the files to only use v1.9.0